metamorph/store/dynamodb/dynamodb.go

```diff
@@ -0,0 +1,461 @@
+package dynamodb
+
+import (
+	"context"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/feature/dynamodb/attributevalue"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb/types"
+	"github.com/bitcoin-sv/arc/metamorph/metamorph_api"
+	"github.com/bitcoin-sv/arc/metamorph/store"
+	"github.com/libsv/go-p2p/chaincfg/chainhash"
+	"github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/ext"
+	"github.com/opentracing/opentracing-go/log"
+	"github.com/ordishs/gocore"
+)
+
+type DynamoDB struct {
+	dynamoCli *dynamodb.Client
+}
+
+const ISO8601 = "2006-01-02T15:04:05.999Z"
```

Comment:
there is already a constant for time format `time.RFC3339`. Maybe worth using that instead? What do you think?

Reply:
Not sure,

From code `RFC3339     = "2006-01-02T15:04:05Z07:00"` So are they really same ? 🤔

Reply:
you are right. It's RFC3339Nano rather.  
see here https://play.golang.com/p/xJuEOoEtHVF